generated/api.github.com.json

```diff
@@ -60,8 +155,25 @@
       "PUT": { "access": "write", "permission": "secrets" }
     },
     "/orgs/{org}/actions/secrets/{secret_name}/repositories/{repository_id}": {
-      "DELETE": { "access": "write", "permission": "secrets" },
-      "PUT": { "access": "write", "permission": "secrets" }
+      "DELETE": { "access": "read", "permission": "metadata" },
+      "PUT": { "access": "read", "permission": "metadata" }
```

Comment:
should be write

Reply:
the docs are wrong:
https://docs.github.com/en/rest/authentication/permissions-required-for-github-apps?apiVersion=2022-11-28#repository-permissions-for-metadata

The `/orgs/{org}/actions/secrets/{secret_name}/repositories/{repository_id}` should not show up there at all. And meta data only has `read`, I'll look into it